### ISSUE TITLE
All snooze events deleted: after around 5 seconds, it goes to Archive

### ISSUE THREAD

**Issue by chelder86** [nylas/nylas-mail#3392]
##### Are there any related issues?
Yes, it seems snooze is giving different problems lately.

##### What operating system are you using?
OSX

##### What version of Nylas Mail are you using?
latest

--

**Bug?**
##### Do you have any third-party plugins installed? If so, which ones?
wunderlist, predawn theme, events calendar and participants display

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
Gmail

##### Is the issue reproducible with a particular attachment, message, signature, etc?

All the snooze and reminders have been deleted

**Comment by chelder86** [comment]
I have deleted the Wunderlist and calendar plugins, and restart Nylas. The same problem still occurs.

I cannot open the log because I got the error "You do not have permission to open the document Nylas-Mail-xxxx.xxxx.log

![screen shot 2017-04-15 at 11 51 25](https://cloud.githubusercontent.com/assets/3020189/25062731/21225ff8-21d2-11e7-9b5e-9a7222a252d8.png)


I'd say that it disappear from the Snooze label as soon as it syncs with Gmail.

**Comment by chelder86** [comment]
It seems reminders are not set either :(

**Comment by chelder86** [comment]
Reminders seem to work well, it just takes some time to appear on the GUI. 

Snoozers does NOT work for me

**Comment by jstejada** [comment]
hi @chelder86 , thanks for the report. We'll look into what's going on. Thanks!

**Comment by chelder86** [comment]
@jstejada it seems fixed with the 2.0 release

**Comment by chelder86** [comment]
Nope. Not working. It was toke longer to be removed from the Snooze folder.

**Comment by chelder86** [comment]
I have deleted the N1 Snooze label from Gmail. It does NOT fix anything. It is not re-created.

**Comment by strika** [comment]
I have a similar problem. Perhaps it's the same, but I'm not sure. When I snooze an email, it doesn't go to "Snoozed" emails and it never returns (e.g. tomorrow morning) to inbox. It's visible only in "All Email". Is it the same problem? Should I open a new issue? Thanks.

**Comment by skandyrocks** [comment]
pretty much the same here - is there any chance of an update soon?
